src/Internal.AspNetCore.Sdk/build/Git.targets

```diff
@@ -19,6 +21,34 @@
     </Sdk_GetGitCommitInfo>
   </Target>
 
+  <Target Name="CreateSourceLink"
+          BeforeTargets="CoreCompile"
+          DependsOnTargets="GenerateCommitHashAttribute" 
+          Condition="'$(SourceLink)'=='' AND '$(GenerateSourceLinkFile)' != 'false' ">
+
+    <Exec Command="git config --get remote.origin.url"
```

Comment:
This invocation should be removed. We sometimes use private forks of repos, and we don't want to disclose this in build artifacts.

You should be able to use `RepositoryUrl` instead, which is pre-defined in our repos. Example: https://github.com/aspnet/Common/blob/dev/Directory.Build.props#L6